docs-ref-services/data-lake-store.md

```diff
@@ -12,41 +12,24 @@ ms.devlang: nodejs
 ms.service: Data Lake Store
 ---
 
-# Azure Data Lake Store modules for Node.js
+# Azure Data Lake Store modules for JavaScript
 
 Azure Data Lake Store is an enterprise-wide hyper-scale repository for big data analytic workloads. Azure Data Lake enables you to capture data of any size, type, and ingestion speed in one single place for operational and exploratory analytics.
 
 ## Management Package
 
 ### Install the npm module
 
-Use npm to install the Azure Data Lake Store modules for Node.js
+Use npm to install the Azure Data Lake Store modules for JavaScript
 
 ```bash
 npm install azure-arm-datalake-store

```

Comment:
`@azure/arm-datalake-store` ?

Reply:
Same as above, couldn't find the newer ARM package for this one. Do you know the new package name for this service?

Reply:
Please revert changes to files for which no counterpart is found in azure-sdk-for-js repo